lib/rules/no-implicit-globals.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @fileoverview Rule to check for implicit global variables and functions.
+ * @author Joshua Peek
```

Comment:
Please include copyright and license info (see http://eslint.org/docs/developer-guide/code-conventions.html#file-format)